kubernetes/frontend.yaml

```diff
@@ -48,11 +56,16 @@ metadata:
     prometheus.io/scrape: "true"
   name: frontend
   namespace: buildbarn
+  labels:
+    app: frontend
 spec:
   ports:
   - port: 8980
     protocol: TCP
     name: grpc
+  - port: 80
```

Comment:
Why do we want to route traffic to port 80 through a service? This is just for Prometheus to scrape, right?